content/uk/docs/contribute/localization_uk.md

```diff
@@ -0,0 +1,123 @@
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Правила перекладу {#правила-перекладу}
+
+* У випадку, якщо у перекладі термін набуває неоднозначності і розуміння тексту ускладнюється, надайте у дужках англійський варіант, наприклад: кінцеві точки (endpoints). Якщо при перекладі термін втрачає своє значення, краще не перекладати його, наприклад: характеристики affinity.
+
+* Співзвучні слова передаємо транслітерацією зі збереженням написання (Service -> Сервіс).
+
+* Реалії Kubernetes пишемо з великої літери: Сервіс, Под, але вузол (node).
+
+* Для слів з великих літер, які не мають трансліт-аналогу, використовуємо англійські слова (Deployment, Volume, Namespace).
+
+* Складені слова вважаємо за власні назви і не перекладаємо (LabelSelector, kube-apiserver).
+
+* Частовживані і усталені за межами K8s слова перекладаємо українською і пишемо з маленької літери (label -> мітка).
+
+* Для перевірки закінчень слів у родовому відмінку однини (-а/-я, -у/-ю) використовуйте [онлайн словник](https://slovnyk.ua/). Якщо слова немає у словнику, визначте його відміну і далі відмінюйте за правилами. Більшість необхідних нам термінів є словами іншомовного походження, які у родовому відмінку однини приймають закінчення -а: Пода, Deployment'а. Докладніше [дивіться тут](https://pidruchniki.com/1948041951499/dokumentoznavstvo/vidminyuvannya_imennikiv).
+
+## Словник {#словник}
+
+English | Українська |
+--- | --- |
+addon | розширення |
+application | застосунок |
+backend | бекенд |
+build | збирати (процес) |
+build | збирання (результат) |
+cache | кеш |
+CLI | інтерфейс командного рядка |
+cloud | хмара; хмарний провайдер |
+containerized | контейнеризований |
+Continuous development | безперервна розробка |
+Continuous integration | безперервна інтеграція |
+Continuous deployment | безперервне розгортання |
+contribute | робити внесок (до проекту), допомагати (проекту) |
+contributor | контриб'ютор, учасник проекту |
+control plane | площина управління |
+controller | контролер |
+CPU | ЦПУ |
+dashboard | дашборд |
+data plane | площина даних |
+default settings | типові налаштування |
+default (by) | за умовчанням |
+Deployment | Deployment |
+deprecated | застарілий |
+desired state | бажаний стан |
+downtime | недоступність, простій |
+ecosystem | сімейство проектів (екосистема) |
+endpoint | кінцева точка |
+expose (a service) | відкрити доступ (до сервісу) |
+fail | відмовити |
+feature | компонент |
+framework | фреймворк |
+frontend | фронтенд |
+image | образ |
+Ingress | Ingress |
+instance | інстанс |
+issue | запит |
+kube-proxy | kube-proxy |
+kubelet | kubelet |
+Kubernetes features | функціональні можливості Kubernetes |
+label | мітка |
+lifecycle | життєвий цикл |
+logging | логування |
+maintenance | обслуговування |
+master | master |
+map | спроектувати, зіставити, встановити відповідність |
+monitor | моніторити |
+monitoring | моніторинг |
+Namespace | Namespace |
+network policy | мережева політика |
+node | вузол |
```

Comment:
When does it make sense to write Node rather than вузол?

Reply:
@anastyakulyk maybe you have answer?

I suppose, only when use it in CLI like `k get nodes`

Reply:
> When does it make sense to write Node rather than вузол?

@sftim we tried our best to keep K8s-specific terms and yet make the translation, not a transliteration. As for the term "вузол" as translation for Node, it is widely used as you can see from Wikipedia, both in Ukrainian (вузол) and in Russian (узел)
https://uk.wikipedia.org/wiki/Kubernetes
https://ru.wikipedia.org/wiki/Kubernetes